src/routers/people.ts

```diff
@@ -74,28 +76,30 @@ router.get('/:category', async (req, res) => {
 }, authorized(UserType.ADMIN));
 
 /**
- * @api {get} /people/<category>
+ * @api {post} /people/<category>
  * @description Creates a new person in the specified category.
  *              Returns the UUID created for that person.
+ *              Requires all associated attributes given in /peopleCategories
  * @apiName CreatePerson
  *
- * @apiError (401) Unauthorized - Must be admin
- * @apiError (404) Person not found in category <category>
+ * @apiError (400) Missing or invalid fields, ...
+ * @apiError (401) Unauthorized

```

Comment:
can u add the @apiParam tag and mention to check the categories api? maybe do this for all of them :D